.azure/pipelines/ci.yml

```diff
@@ -7,22 +7,716 @@ pr:
-    agentOs: macOS
-- template: jobs/default-build.yml
-  parameters:
-    jobName: Linux_Build
-    jobDisplayName: "Build and test : Linux"
-    agentOs: Linux
+# Unix test jobs only run on public CI builds
+- ${{ if eq(variables['System.TeamProject'], 'public') }}:
+  - template: jobs/default-build.yml
+    parameters:
+      jobName: MacOs_Build
+      jobDisplayName: "Build and test : MacOS"
+      agentOs: MacOS
+  - template: jobs/default-build.yml
+    parameters:
+      jobName: Linux_Build
+      jobDisplayName: "Build and test : Linux"
+      agentOs: Linux
+
+- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+  - job: Windows_SharedFx
+    displayName: Build Windows x64/x86 SharedFx
+    dependsOn: Windows_Build
+    timeoutInMinutes: 90
+    workspace:
+      clean: all
+    pool:
+      name: NetCoreInternal-Pool
+      queue: BuildPool.Server.Amd64.VS2017
+    variables:
+      _SignType: real
+      JAVA_HOME: $(Agent.BuildDirectory)\.tools\jdk
+    steps:
+    - checkout: self
+      clean: true
+    - task: NodeTool@0
+      displayName: Install Node 10.x
+      inputs:
+        versionSpec: 10.x
+    - powershell: ./eng/scripts/InstallJdk.ps1 '11.0.1'
+      displayName: Install JDK 11
+    - task: NuGetCommand@2
+      displayName: 'Clear NuGet caches'
+      condition: succeeded()
+      inputs:
+        command: custom
+        arguments: 'locals all -clear'
+    - task: MicroBuildSigningPlugin@2
+      displayName: Install MicroBuild Signing plugin
+      inputs:
+        signType: $(_SignType)
+        zipSources: false
+        feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
+    - task: DownloadBuildArtifacts@0
+      displayName: Download artifacts
+      inputs:
+        artifactName: artifacts-Windows-Release
+        downloadPath: $(Build.StagingDirectory)/downloaded_artifacts/
+    - task: CopyFiles@2
+      displayName: Copy artifacts to .deps/
+      inputs:
+        sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-Windows-Release/
+        contents: '**'
+        targetFolder: $(Build.SourcesDirectory)/.deps/
+    - script: .\build.cmd
+              -ci
+              /p:SignType=$(_SignType)
+              $(BuildScriptArgs)
+              $(SharedFxArgs)
+              /p:SharedFxRID=win-x64
+              /bl:artifacts/logs/SharedFx-win-x64.binlog
+      env:
+        PB_PackageVersionPropsUrl: $(PB_PackageVersionPropsUrl)
+        PB_AssetRootUrl: $(PB_AssetRootUrl)
+        PB_RestoreSource: $(PB_RestoreSource)
+        PB_PublishBlobFeedKey: $(PB_PublishBlobFeedKey)
+      displayName: Build win-x64 SharedFX
+    - powershell: eng\scripts\KillProcesses.ps1
+      displayName: Kill processes
+      condition: always()
+    - script: .\build.cmd
+              -ci
+              /p:SignType=$(_SignType)
+              $(BuildScriptArgs)
+              $(SharedFxArgs)
+              /p:SharedFxRID=win-x86
```

Comment:
Hmm the display name says x64 but this RID says x86? I think the name got flipped with the one below this.

Reply:
The display name for this step is below, on line 125. The display name on line 109 is for the first build.cmd invocation